building deb package on stretch broken (debhelper version changed)

ping @tianon since you are a maintainer of the debhelper package maybe you know what sepcifically changed

so it seems the debhelper version on stretch has been bumped (debhelper (9.20160306)) https://packages.debian.org/stretch/debhelper http://metadata.ftp-master.debian.org/changelogs/main/d/debhelper/debhelper_9.20160306_changelog

so this leads to errors when trying to build the docker-engine debs:

```
chmod: cannot access '../../hack/make/.build-deb/docker-engine/usr/share/man/man5/config-json.5': No such file or directory
chmod: cannot access '../../hack/make/.build-deb/docker-engine/usr/share/man/man8/docker-daemon.8': No such file or directory
dh_compress: chmod a-x ../../hack/make/.build-deb/docker-engine/usr/share/doc/docker-engine/README.md ../../hack/make/.build-deb/docker-engine/usr/share/doc/docker-engine/changelog ../../hack/make/.build-deb/docker-engine/usr/share
/doc/docker-engine/changelog.Debian ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-attach.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-build.1 ../../hack/make/.build-deb/docker-engine/usr/share/
man/man1/docker-commit.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-cp.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-create.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docke
r-diff.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-events.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-exec.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-export.1 ../
../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-history.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-images.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-import.1 ../../hack/m
ake/.build-deb/docker-engine/usr/share/man/man1/docker-info.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-inspect.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-kill.1 ../../hack/make/.build-de
b/docker-engine/usr/share/man/man1/docker-load.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-login.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-logout.1 ../../hack/make/.build-deb/docker-engi
ne/usr/share/man/man1/docker-logs.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-network-connect.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-network-create.1 ../../hack/make/.build-deb/docker
-engine/usr/share/man/man1/docker-network-disconnect.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-network-inspect.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-network-ls.1 ../../hack/make/.b
uild-deb/docker-engine/usr/share/man/man1/docker-network-rm.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-pause.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-port.1 ../../hack/make/.build-deb/
docker-engine/usr/share/man/man1/docker-ps.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-pull.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-push.1 ../../hack/make/.build-deb/docker-engine/usr/
share/man/man1/docker-rename.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-restart.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-rm.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man
1/docker-rmi.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-run.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-save.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-search.1
../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-start.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-stats.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-stop.1 ../../hack/mak
e/.build-deb/docker-engine/usr/share/man/man1/docker-tag.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-top.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-unpause.1 ../../hack/make/.build-deb/do
cker-engine/usr/share/man/man1/docker-update.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-version.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-volume-create.1 ../../hack/make/.build-deb/dock
er-engine/usr/share/man/man1/docker-volume-inspect.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-volume-ls.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-volume-rm.1 ../../hack/make/.build-deb/
docker-engine/usr/share/man/man1/docker-volume.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker-wait.1 ../../hack/make/.build-deb/docker-engine/usr/share/man/man1/docker.1 ../../hack/make/.build-deb/docker-engine/usr/s
hare/man/man5/Dockerfile.5 ../../hack/make/.build-deb/docker-engine/usr/share/man/man5/config-json.5 ../../hack/make/.build-deb/docker-engine/usr/share/man/man8/docker-daemon.8 returned exit code 1
debian/rules:40: recipe for target 'binary' failed
make: *** [binary] Error 1
dpkg-buildpackage: error: debian/rules binary gave error exit status 2
```

--- a/hack/make/build-deb
+++ b/hack/make/build-deb
@@ -65,7 +65,7 @@ set -e
 			echo 'ENV DOCKER_EXPERIMENTAL 1' >> "$DEST/$version/Dockerfile.build"
 		fi
 		cat >> "$DEST/$version/Dockerfile.build" <<-EOF
-			RUN ln -sfv hack/make/.build-deb debian
+			RUN mv -v hack/make/.build-deb debian
 			RUN { echo '$debSource (${debVersion}-0~${suite}) $suite; urgency=low'; echo; echo '  * Version: $VERSION'; echo; echo " -- $debMaintainer  $debDate"; } > debian/changelog && cat >&2 debian/changelog
 			RUN dpkg-buildpackage -uc -us
 		EOF

